### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by oyelakinG9**
Validation: To check if the email was written in lower case.

### CODE REVIEW

File: js/my_portfolio.js [hunk 1]
```diff
@@ -223,4 +223,49 @@ function render() {
...
+
+  const emailRegex = /[A-Z]/g;
+
+  if (emailValue.match(emailRegex)) {
+    showError(`Email field doesn't allow capital letters. It should be ${emailValue.toLowerCase()}`);
+    return false;
+  }
+
+  return true;
+}
```
Comment (julie-ify) [inline js/my_portfolio.js:255]: - [ ] Currently, I entered my email in all uppercase characters and I was able to send an email which is not what is required. It is required to display an error message when a user enters his/her email in uppercase which tells them to enter their email in lowercase characters. Please try fixing this error in other to meet the requirement 😃.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!

Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️  
- Descriptive commit messages ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/my_portfolio.js (hunk 1 above)
Comment (oyelakinG9) [reply js/my_portfolio.js:255]: Hi @julie-ify, It's working perfectly, we are to generate a function to create an error message when a user submits the form that contains an upperCase character.

**Review by oxenprogrammer — APPROVED** [review]
Hi @oyelakinG9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

Looks like the issue was resolved here, so I have nothing else to say other than best of luck with the rest of your projects!

Keep rocking 🚀!

**Pull request merged**